diff --git a/node_modules/ipfs-utils/src/http.js b/node_modules/ipfs-utils/src/http.js
index 0180fdf..40153f9 100644
--- a/node_modules/ipfs-utils/src/http.js
+++ b/node_modules/ipfs-utils/src/http.js
@@ -1,7 +1,7 @@
 /* eslint-disable no-undef */
 'use strict'
 
-const { fetch, Request, Headers } = require('./http/fetch')
+const { Request, Headers } = require('./http/fetch')
 const { TimeoutError, HTTPError } = require('./http/error')
 const merge = require('merge-options').bind({ ignoreUndefined: true })
 const { URL, URLSearchParams } = require('iso-url')
