diff --git a/node_modules/ipfs-utils/src/fetch.js b/node_modules/ipfs-utils/src/fetch.js
index a941d92..fe4fba4 100644
--- a/node_modules/ipfs-utils/src/fetch.js
+++ b/node_modules/ipfs-utils/src/fetch.js
@@ -9,10 +9,6 @@
 const { isElectronMain } = require('./env')
 
 // use window.fetch if it is available, fall back to node-fetch if not
-let impl = 'native-fetch'
-
-if (isElectronMain) {
-  impl = 'electron-fetch'
-}
-
-module.exports = require(impl)
+module.exports = (isElectronMain) ?
+  require('electron-fetch') : 
+  require('native-fetch')
diff --git a/node_modules/ipfs-utils/src/http/fetch.js b/node_modules/ipfs-utils/src/http/fetch.js
index 1a27129..1e7e940 100644
--- a/node_modules/ipfs-utils/src/http/fetch.js
+++ b/node_modules/ipfs-utils/src/http/fetch.js
@@ -8,15 +8,10 @@
  * @property {globalThis.Headers} fetchImpl.Headers
  */
 
-let implName = './fetch.node'
-
-if (typeof XMLHttpRequest === 'function') {
-  // Electron has `XMLHttpRequest` and should get the browser implementation
-  // instead of node.
-  implName = './fetch.browser'
-}
-
 /** @type {fetchImpl} */
-const fetch = require(implName)
+const fetch = 
+  (typeof XMLHttpRequest === 'function') ? 
+    require('./fetch.browser') : 
+    require('./fetch.node') 
 
 module.exports = fetch
